src/components/Bookmark/Container/container.module.scss

```diff
@@ -0,0 +1,48 @@
+@import '../../../assets/styles/variables';
+
+.bookmark-container {
+	display: grid;
+	grid-template-columns: 1fr;
+	gap: 2rem;
+
+	&_theme {
+		&_light {
+			& .bookmark-container__new-group-button {
+				color: themed('light', 'bookmarkGroupNewButtonColor');
```

Comment:
Overriding sounds good because it does not involve JavaScript, but implementing CSS variables will require changing many files across the project. I will do it tomorrow, after the demo.

Reply:
Done. You are right, CSS variables are much easier to use in components